keras/backend/common/variables_test.py

```diff
@@ -140,6 +141,26 @@ def test_autocasting(self):
         with AutocastScope("float16"):
             self.assertEqual(backend.standardize_dtype(v.value.dtype), "int32")
 
+    def test_standardize_dtype(self):
```

Comment:
This may call for a different factoring --

1. use a `parameterized` decorator to iterate over `ALLOWED_DTYPES`
2. use `if backend.backend() == ...` and `if dtype in (...):` to do `self.skipTest(f"{backend.backend()} does not support dtype {dtype}")`